include/SFML/Window/Clipboard.hpp

```diff
@@ -43,27 +43,27 @@ class SFML_WINDOW_API Clipboard
 {
 public:
 
-	////////////////////////////////////////////////////////////
-	/// \brief Get the content of the clipboard as string data
-	///
-	/// This function returns the content of the clipboard
-	/// as a string.
-	///
-	/// \return Current content of the clipboard
-	///
-	////////////////////////////////////////////////////////////
-	static String getString();
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the content of the clipboard as string data
+    ///
+    /// This function returns the content of the clipboard
+    /// as a string.
+    ///
```

Comment:
I haven't added support for other forms of data because I haven't thought of a proper API to implement it and also ensure it behaves the same on all of SFML's supported platforms. I don't think images can be copied to the clipboard on Android; maybe on newer versions? I don't know, my phone is old (4.4.2).